.github/workflows/build.yml

```diff
@@ -3,9 +3,13 @@ name: CI
 on:
   workflow_dispatch:
   push:
-    branches: [main]
+    branches:
+      - 'main'
+      - 'v1.0'
   pull_request:
-    branches: [main]
+    branches:
+      - 'main'
+      - 'v1.0'
```

Comment:
I think this can be reverted now that we have tested it's working

Reply:
So there's two pieces: `push` and `pull_request`.

1. `push` is redundant, because #639 is created to merge with `main`, so this workflow should run every time we merge a PR into the `v1.0` branch

2. `pull_request` we could leave to run build/tests on every individual v1 PR, but it won't publish test results to gh-pages. That might be ok.

Just thinking out loud, maybe this is fine.